### PR TITLE
Resources needed to support order-select hook drug-drug interactions.

### PR DESCRIPTION
**Pull request by plarocque4**
Add resources needed to support medication interactions with immunosuppressive drugs and the order-select hook.

### CODE REVIEW

**Comment by plarocque4** [comment]
Test with order-select branches for CRD, CDS-Library, crd-request-generator and test-ehr.

File: fhirResourcesToLoad/medication_methotrexate.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            "code": "Tab"
+          }
+        }
+      }
+    ],
+    "batch": {
+      "lotNumber": "12345",
+      "expirationDate": "2026-06-23"
+    }
+  }
```
Comment (yuliumitre) [inline fhirResourcesToLoad/medication_methotrexate.json:51]: Using FHIR validator, the following validation errors are found: 

Validation errors:
MedicationStatement: dom-3: 'If the resource is contained in another resource, it SHALL be referred to from elsewhere in the resource or SHALL refer to the containing resource' 'If the resource is contained in another resource, it SHALL be referred to from elsewhere in the resource or SHALL refer to the containing resource' ( (unmatched: med-methotrexate)) on line 1. Jump to error.
MedicationStatement.contained.ofType(Medication).ingredient[0].strength.denominator: Unknown Code http://terminology.hl7.org/CodeSystem/v3-orderableDrugForm#Tab in http://terminology.hl7.org/CodeSystem/v3-orderableDrugForm for 'http://terminology.hl7.org/CodeSystem/v3-orderableDrugForm#Tab' on line 44. Jump to error.

File: fhirResourcesToLoad/o2teddy_medicationstatement_methotrexate.json [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+                    "strength": {
+                        "numerator": {
+                            "value": 7.5,
+                            "system": "http://unitsofmeasure.org",
+                            "code": "mg"
+                        },
+                        "denominator": {
+                            "value": 1,
+                            "system": "http://terminology.hl7.org/CodeSystem/v3-orderableDrugForm",
+                            "code": "Tab"
```
Comment (yuliumitre) [inline fhirResourcesToLoad/o2teddy_medicationstatement_methotrexate.json:46]:
```suggestion
                            "code": "Tab"  "TAB"
```

File: fhirResourcesToLoad/medication_methotrexate.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        "strength": {
+          "numerator": {
+            "value": 7.5,
+            "system": "http://unitsofmeasure.org",
+            "code": "mg"
+          },
+          "denominator": {
+            "value": 1,
+            "system": "http://terminology.hl7.org/CodeSystem/v3-orderableDrugForm",
+            "code": "Tab"
```
Comment (yuliumitre) [inline fhirResourcesToLoad/medication_methotrexate.json:42]:
```suggestion
            "code": "TAB"
```

File: fhirResourcesToLoad/o2teddy_medicationstatement_methotrexate.json (hunk 2 above)
Comment (plarocque4) [reply fhirResourcesToLoad/o2teddy_medicationstatement_methotrexate.json:46]: Thanks, will update.

File: fhirResourcesToLoad/medication_methotrexate.json (hunk 3 above)
Comment (plarocque4) [reply fhirResourcesToLoad/medication_methotrexate.json:42]: Thanks, will update.

**Review by yuliumitre — APPROVED** [review]

**Pull request merged**